module/Activity/src/Activity/Service/ActivityQuery.php

```diff
@@ -121,30 +121,31 @@ public function getAllProposals()
 
     /**
      * Get an array that states whether a language is available for
-     * the provided $activity
+     * the provided $activity.
      *
      * @param ActivityModel $activity
+     *
      * @return array
      */
     public function getAvailableLanguages($activity)
     {
         return ['nl' => !is_null($activity->getName()->getValueNL()),
-            'en' => !is_null($activity->getName()->getValueEN())];
+            'en' => !is_null($activity->getName()->getValueEN()), ];
```

Comment:
There are now weird cases where commas are added while it does not fit the actual structure of the array. This should be fixed.

Reply:
For multiline arrays I think it's better to have them.

Reply:
Yeah, but then we should change the layout of the array (in this specific case that would mean putting the `];` on a new line)